main.py

```diff
@@ -0,0 +1,141 @@
+                d = f.readlines()
+                f.seek( 0 )
+                for i in d:
+                    if i != category_name + '\n' and i != category_name:
+                        f.write( i )
+                f.truncate()
+                f.close()
+                print("Category is removed")
+                break
+            else:
+                print("Category doesn't exist")
+                continue
+
+    def menu(self):
+        text_menu = """ 
+                Press 1 if you want to add category
+                Press 2 if you want to remove category
+                Press 3 if you want to see categories
+                Press 4 if you want to exit from edit
+                """
+        while True:
+            choose = raw_input(text_menu)
+            if choose.isdigit():
+                if int(choose) == 1:
+                    self.check_file()
+                    self.add_category()
+                elif int(choose) == 2:
+                    self.remove_category()
+                elif float(choose) == 3:
+                    print(self.categories)
+                elif float(choose) == 4:
+                    print("Exit")
+                    break
+                else:
+                    print("Not correct value, try again")
+            else:
+                print("Not correct value, try again")
+                continue
+
+
+
+class ExpenseManager:
+
+    def expense(self):
+        #function to return details of expense
+        while True:
+            name = raw_input("Enter name of category")
+            if name not in self.categories:
+                print("Category doesn't exists")
+                continue
+            else:
+                amount = float("Enter an amount")
+                day_expense = int(raw_input( "Enter a day"))
+                month_expense = int(raw_input( "Enter a month"))
+                year_expense = int(raw_input( "Enter a year"))
+                date_expense = datetime( year_expense,month_expense,day_expense, tzinfo=TZ()).isoformat(' ')
+                return name, amount,date_expense
+
+category_object = CategoryManager()
+category_object.check_file()
+
+class Menu:
+
+    def menu_content(self):
+        while True:
+            print("""
+                    1.Press 1 If you want to Manage of Category
+                    2.Press 2 If you want to Manage to expense 
+                    3.Press 3 Exit from Manager
+                    """)
+            choose = raw_input("Enter a number")
+            if choose.isdigit():
+                if int(choose) == 1:
+                    category_object.menu()
+                elif int(choose) == 2:
+                    pass
+                elif float(choose) == 3:
+                    print("Exit")
+                    sys.exit()
+                else:
+                    print("Not correct value, try again")
+            else:
+                print("Not correct value, try again")
```

Comment:
1. Maybe `Incorrect` would be more suitable
2. This text is used in many places - please use variable